vtn/src/main/java/com/lftechnology/vtn/adapter/LocalDateAdapter.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.vtn.adapter;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class is used to convert date in string format to LocalDate
+ *
+ * @author shankar ghimire
+ */
+
+public class LocalDateAdapter extends XmlAdapter<String, LocalDate> {
+
+    @Override
+    public String marshal(LocalDate date) {
```

Comment:
why are we marshalling and unmarshalling using different format?

Reply:
its because response can have two different date format: one with date and time and another with date only. We created adapter for each format.